public/pages/AnomalyCharts/utils/constants.ts

```diff
@@ -90,6 +92,7 @@ export const DEFAULT_ANOMALY_SUMMARY = {
   minConfidence: 0,
   maxConfidence: 0,
   lastAnomalyOccurrence: '-',
+  contributions: '-'
```

Comment:
I'm confused about this part - are we showing feature contributions for all anomalies in the chart as a metric somehow? The above fields are all shown as metric-like headers above the anomaly chart.

Reply:
This is not used, forgot to clean it